core/src/main/java/org/opensearch/sql/ast/expression/LiteralList.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.ast.expression;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.opensearch.sql.ast.AbstractNodeVisitor;
+
+/**
+ * Expression node that includes a list of Expression nodes.
+ */
+@ToString
+@EqualsAndHashCode(callSuper = false)
+@AllArgsConstructor
+public class LiteralList extends UnresolvedExpression {
+  @Getter
+  private List<Literal> literalList;
+
+  @Override
+  public List<Literal> getChild() {
+    return ImmutableList.copyOf(literalList);
+  }
+
+  @Override
+  public <R, C> R accept(AbstractNodeVisitor<R, C> nodeVisitor, C context) {
+    return nodeVisitor.visitLiteralList(this, context);
+  }
+
+  @Override
+  public String toString() {
+    return literalList.stream().map(String::valueOf).collect(Collectors.joining(", "));
```

Comment:
Is the space required? Can it also be `","`?

Reply:
No, but it makes string representation more readable.

Reply:
I misread this code.  I thought it was part of the parsing, now I see that it's a join. 
Please ignore this

Reply:
Yes, it could be, but a space make string representation of the lest more readable:
`"one","two","three"` vs `"one", "two", "three"`.